src/styles/index.scss

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.minutes {
+  position: absolute;
+  bottom: 50%;
+
+  height: 20vmin;
+  width: 3vmin;
+  background-color: #0700ff;
+
+  animation: rotate-bottom 3600s steps(60, end) infinite forwards;
```

Comment:
The animation duration for `.minutes` is very long (3600s). Make sure that this is intentional and serves the design purpose. If it's meant to represent a real stopwatch, the duration should match the real-world time it's emulating.